docs-src/multiple-ns.md

```diff
@@ -0,0 +1,32 @@
+# Routes in multiple namespaces
+
+With the Service APIs, a single Gateway can target routes across multiple
+namespaces. 
+
+This guide assumes that you have installed Service APIs CRDs and a conformant
+controller.
+
+In the following example:
+
+- `acme-lb` GatewayClass: The GatewayClass responsible for satisfying Gateway
+  and Route resources.
+- `multi-ns-gateaway` Gateway: The Gateway is configured with a single listener
+  on port 80 which selects routes from **All namespaces** which have the label
+  `product: baz`. Notice how the `routes.namespaces.from` field in the listener
+  is set to `All`.
+- `foo-ns` and `bar-ns` Namespaces: These are two namespaces in which we route
```

Comment:
I'm not exactly sure where this belongs, but it would probably be good to call attention to the fact that this example creates namespaces and explicitly installs resources in them. If a user is trying to keep all their examples in a single namespace, that will not work here. Similarly, if they happen to be using namespaces with these names that could be problematic. Especially if they happen to cleanup demo resources with `kubectl delete -f`. 

Maybe to be extra safe resources like namespaces should have a clear association with this project? Something like `service-apis-example-ns1`, `service-apis-example-ns2` is overly verbose but also unlikely to conflict with any existing namespaces. Maybe there's a less verbose option that has a similar effect of clearly being tied to an example from this project?

Reply:
Opted to change the name to not conflict with anything that already exists.
I'm not explicitly calling it out because it seems self-explanatory. Is that okay?